dso/model.py

```diff
@@ -355,6 +357,12 @@ def key(self) -> str:
         return self.finding.key
 
 
+@dataclasses.dataclass(frozen=True)
+class SastFinding(Finding):
```

Comment:
We'll also have to add the `key` property here since it's required for objects of `CustomRescoring` class ([see](https://github.com/gardener/cc-utils/blob/fd2771aff40506c10094136cc3045faaf4eb9988/dso/model.py#L428)).